### PR TITLE
Some feedbacks from our PM

### PR DESCRIPTION
**Pull request by NeOMakinG**

### CODE REVIEW

File: _dev/front/js/components/List/List.vue [hunk 1]
```diff
@@ -27,51 +27,56 @@
...
-          <button
-            @click="toggleShare(list.id_wishlist, list.shareUrl)"
-            v-if="list.default"
-          >
-            <i class="material-icons">share</i>
-          </button>
+          <div class="wishlist-list-item-right">
+            <button
+              class="wishlist-list-item-actions"
+              @click.stop="togglePopup(list.id_wishlist)"
```
Comment (PierreRambaud) [inline _dev/front/js/components/List/List.vue:45]:
```suggestion
              @click.prevent="togglePopup(list.id_wishlist)"
```
Should be enough :thinking:

File: _dev/front/js/components/Product/Product.vue [hunk 2]
```diff
@@ -190,6 +213,16 @@
...
     },
+    computed: {
+      isDisabled() {
+        if (this.product.customization_required) {
+          return false;
+        } if (!this.product.add_to_cart_url) {
+          return true;
+        }
+        return false;
+      },
```
Comment (PierreRambaud) [inline _dev/front/js/components/Product/Product.vue:224]:
```suggestion
        if (this.product.customization_required) {
          return false;
        } 

        return !this.product.add_to_cart_url;
      },
```

File: _dev/front/js/components/Product/Product.vue [hunk 3]
```diff
@@ -204,49 +237,53 @@
...
+          try {
+            const response = await fetch(
+              `${this.product.add_to_cart_url}&action=update`,
+              {
+                headers: {
+                  'Content-Type':
+                    'application/x-www-form-urlencoded; charset=UTF-8',
+                  Accept: 'application/json, text/javascript, */*; q=0.01',
+                },
+              },
```
Comment (PierreRambaud) [inline _dev/front/js/components/Product/Product.vue:250]: :fearful:

File: _dev/front/js/components/List/List.vue (hunk 1 above)
Comment (NeOMakinG) [reply _dev/front/js/components/List/List.vue:45]: Nop, it prevent the event sent by the element itself, not the event from the parent element

**Pull request merged**